doc/xcodebuild.txt

```diff
@@ -18,6 +18,9 @@ xcodebuild. By default, xcodebuild.vim passes the `--color` flag to xcpretty.
 ==============================================================================
 CONFIGURATION                             *xcodebuild-configuration*
 
+* Set the runner to use for the xcodebuild command (uses ! by default)
+ let g:xcodebuild_runner = 'VtrSendCommandToRunner'
```

Comment:
I was literally going to ask you about this tomorrow. Fantastic note, I'll add that to this and the README.